style.css

```diff
@@ -0,0 +1,228 @@
+/* My body config */
+body {
+  margin: 0;
+  padding: 0;
+  background: white;
+  box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+  height: auto;
+}
+
+/* initialize the .header style: sizes, colors and fonts */
+.header {
+  padding-top: 1px;
+  height: 682px;
+  margin: 0;
+  color: white;
+  background-color: black;
+  width: 100%;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+}
+
+/* using flexbox for nav tag */
+.nav {
+  margin-bottom: -100px;
+  width: 100%;
+  position: fixed;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #3c3a39;
+}
+
+.my-logo {
+  margin-left: 24px;
+}
+
+.burger-menu {
+  font-size: 30px;
+  margin: 10px 35px 10px 10px;
+}
+
+/* initialize the .main and use flexbox for all his childs */
+.main {
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  margin-left: 24px;
+}
+
+/* initialize the .h,h-1,h-2,h-3: sizes, colors, and fonts */
+.h-1 {
+  width: 327px;
+  height: 124px;
+  color: #ff6b00;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  margin-top: 190px;
+}
+
+.h-2 {
+  color: #fff5e1;
+  width: 327px;
+  height: 27px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+}
+
+.para-1 {
+  color: white;
+  width: 327px;
+  height: 123px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+/* -- add style to div that contain the  img -- */
+
+.div-img {
+  display: flex;
+  position: absolute;
+  top: 250px;
+  bottom: 212px;
+  margin-left: 210px;
+  justify-content: end;
+  float: right;
+}
+
+.img-1 {
+  margin-top: 70px;
+  margin-left: 35px;
+  width: 100%;
+  height: 410px;
+}
```

Comment:
- [ ] Good work including the background image in your headline ✔️. However, please kindly make sure to apply the correct position for your image, currently, it is a few extra pixels left 🤓.

This is your design 👇🏽

![design_2](https://user-images.githubusercontent.com/116701485/235018804-7a76ea56-7c3d-4b61-9ec7-32ef479e95d0.png)

This is the Figma design 👇🏽 (almost there 😅)

![figma_2](https://user-images.githubusercontent.com/116701485/235018854-a84d1ada-697e-4d0f-ac19-2ac79a884bae.png)